jarviscli/plugins/voice_control.py

```diff
@@ -15,27 +17,50 @@
         'voice_control_requirements (install portaudio + re-run setup.sh)']
 
 
+def has_internet():
```

Comment:
+1
We could probably even move this function to JarvisAPI so it's available to all plugins.
And/or use this to (finally) implement ``@require(network=True)`` ;).

Reply:
yep, that sounds like a great idea :)

Reply:
@pnhofmann can you guide me into where the plugin requirements are checked? I want to help implement the network requirements,  but I'm not sure where to start. My best guess was in plugin.py in `_plugin_run_with_network_error` because the docstrings said that the method is auto run when `require(network=True)` is provided.